plugins/languages/treesitter/treesitter.nix

```
@@ -158,7 +167,7 @@ helpers.neovim-plugin.mkNeovimPlugin config {
     auto_install = false;
     ensure_installed = [ "all" ];
     ignore_install = [ "rust" ];
-    parser_install_dir = "$XDG_DATA_HOME/nvim/treesitter";
+    parser_install_dir.__raw = "vim.fs.joinpath(vim.fn.stdpath('data'), 'site')";
```

```suggestion
```

No need to include default values in the example.

Could also do this:

```suggestion
    parser_install_dir.__raw = "vim.fs.joinpath(vim.fn.stdpath('data'), 'treesitter')";
```
But IMO it distracts from more useful parts of the example. Especially since the user _also_ needs to add the path to the rtp and we can't really show that in the example

---

We automatically add that path to rtp in the extraConfig, though. Do you want me to leave that example as it was prior to show how to set to the XDG path then?

---

I don't mind having an example, but it should be different to the default.

Ideally examples should be a useful "real world" example. If that's not possible an obvious "psudo" example that users know not to copy verbatim is ok too. But we don't need examples just for the sake of having them.

If we add the setting to the rtp in extraConfig then we can probably drop the note I suggested in the option desc too?